crates/blockchain-tree/src/blockchain_tree.rs

```diff
@@ -1601,7 +1600,6 @@ mod tests {
             #[cfg(not(feature = "optimism"))]
             let receipts_root = calculate_receipt_root(&receipts);
 
-            #[cfg(feature = "optimism")]
```

Comment:
this is incorrect, please remove this. it should only happen when the optimism feature is enabled.